libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -285,17 +285,17 @@
         "limit": 0
     },
     "dashboard.bucket.metric_segment_by_warning": {
-        "value": "To add additional measure, remove <span class=\"attr-field-icon\"></span> from <span class=\"stack-by\">segment by</span>",
+        "value": "To add additional measure, remove {icons} from <span class=\"stack-by\">segment by</span>",
```

Comment:
please put below some info for translators how to deal with this {icons} string

Reply:
Done